tests/test_web.py

```diff
@@ -298,8 +298,8 @@ def test_create_user_comment(self) -> None:
         if not instance_config.get('registration'):
             return None
 
-        user = self.client.create_user(name='test_Name', login='test_Login',
-                                       organisation='test Organization', email='test@testorg.local')
+        user = self.client.create_user(name='test Name', login='alan',
+                                       organisation='test Organization', email='test@testorg.lu')
```

Comment:
This is very much a detail, but I avoid using any TLD other than `.local` because at some point, a system will be configured to send emails and this is going to be a problem.

Reply:
I prefer `.local`: https://en.wikipedia.org/wiki/.local

`example.com` doesn't have a MX record, but it could one day.

Reply:
just say you comment here. Anyway, it's fine now ;-)